routes/impersonation.ts

```
@@ -0,0 +1,12 @@
+import express from "express";
+const router = express.Router();
+const authorizeRoles = require("../middlewares/authorizeRoles");
+const { SUPERUSER } = require("../constants/roles");
+import authenticate from "../middlewares/authenticate";
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)